README.md

```diff
@@ -31,8 +31,17 @@
 
 At the same time, `mamba` utilizes the same command line parser, package installation and deinstallation code and transaction verification routines as `conda` to stay as compatible as possible.
 
-Mamba is part of a bigger ecosystem to make scientific packaging more sustainable. You can read our [announcement blog post](https://medium.com/@QuantStack/open-software-packaging-for-science-61cecee7fc23).
-The ecosystem also consists of `quetz`, an open source `conda` package server and `boa`, a fast `conda` package builder.
+`mamba` is part of the [conda-forge](https://conda-forge.org/) ecosystem, which also consists of `quetz`, an open source `conda` package server and `boa`, a fast `conda` package builder.
+
+You can read our [announcement blog post](https://medium.com/@QuantStack/open-software-packaging-for-science-61cecee7fc23).
+
+## micromamba
+
+`micromamba` is the statically linked version of `mamba`.
+
+It can be installed as a standalone executable without any dependencies, making it a perfect fit for CI/CD pipelines and containerized environments.
```

Comment:
It may be nice to also give examples of when `mamba` would be preferable to `micromamba` (if there are any?)

Reply:
I cannot really think of any benefits of using `mamba` over `micromamba` for now (except retro-compatibility for end-users' application).

Reply:
makes sense! I wonder how that can be more clearly reflected in the docs (maybe it isn't important)

Reply:
By being dynamically linked, mamba benefits from the updates of its dependencies, while someone using micromamba must wait for a new release of micromamba to get the updates.

I would say micromamba targets CI and "non transient", not sensible environments (like developers machine) while mamba is more for production environments. Also if you want ot use the Python API, it might be better to install mamba (otherwise it's like installing libmamba twice).

Reply:
> By being dynamically linked, mamba benefits from the updates of its dependencies, while someone using micromamba must wait for a new release of micromamba to get the updates.
> 
> I would say micromamba targets CI and "non transient", not sensible environments (like developers machine) while mamba is more for production environments. Also if you want ot use the Python API, it might be better to install mamba (otherwise it's like installing libmamba twice).

Cool! Would be great for this to make it into the docs.

Reply:
Added with f477533, let me know if this is clear to you.

Reply:
@lucascolley: is this clear to you?

Reply:
looks fine to me, thanks!